### PR TITLE
Adding illustrations for the K8s Top 10

### PR DESCRIPTION
**Pull request by gmaran23**
Adding Gifs to illustrate a Top 10 vulnerability, and present a few mitigations.

To get a perspective of how this PR would look like, please review https://github.com/gmaran23/www-project-kubernetes-top-ten/blob/main/2022/en/src/K01-insecure-workload-configurations.md

### CODE REVIEW

File: 2022/en/src/K05-inadequate-logging.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 A Kubernetes environment has the ability to generate logs at a variety of levels from many different components. When logs are not captured, stored, or actively monitored attackers have the ability to exploit vulnerabilities while going largely undetected. The lack of logging and monitoring also presents challenges during incident investigation and response efforts. 
 
+![Inadequate Logging - Illustration](/assets/images/K05-2022.gif)
+
```
Comment (szh) [inline 2022/en/src/K05-inadequate-logging.md:6]: I'm not entirely sure what this illustration is showing. Can you please elaborate?

File: 2022/en/src/K05-inadequate-logging.md [hunk 2]
```diff
@@ -34,6 +36,8 @@ Managed Kubernetes providers, including AWS, Azure, and GCP provide optional acc
 
 As outlined above, there is no shortage of logging mechanisms available within the Kubernetes ecosystem. A robust security logging architecture should not only capture relevant security events, but also be centralized in a way that is queryable, long term, and maintains integrity.
 
+![Inadequate Logging - Mitigations](/assets/images/K05-2022-mitigation.gif)
+
```
Comment (szh) [inline 2022/en/src/K05-inadequate-logging.md:40]: Same here. Can you provide more details?

File: 2022/en/src/K06-broken-authentication.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ## Overview
 Authentication in Kubernetes takes on my many forms and is extremely flexible. This emphasis on being highly configurable makes Kubernetes work in a number of different environments but also presents challenges when it comes to cluster and cloud security posture.
 
+![Broken Authentication - Illustration](/assets/images/K06-2022.gif)
```
Comment (szh) [inline 2022/en/src/K06-broken-authentication.md:4]: Do you mean OIDC as opposed to ODIC?

File: 2022/en/src/K06-broken-authentication.md [hunk 4]
```diff
@@ -30,6 +31,8 @@ Service account (SA) tokens can be presented to the Kubernetes API as an authent
 
 ***Authenticate users and external services using short-lived tokens:*** All authentication tokens should be as short-lived as tolerable. This way if (and when) a credential is leaked, it is possible that it may not be replayed in the time necessary to compromise the account. 
 
+![Broken Authentication - Mitigations](/assets/images/K06-2022-mitigation.gif)
```
Comment (szh) [inline 2022/en/src/K06-broken-authentication.md:34]: It looks like some of the recommendations in the illustration are the opposite of the markdown text...

**Review by szh — CHANGES_REQUESTED** [review]
These are amazing! I love the aesthetics and they could be really helpful in presentations. I left a few comments on individual illustrations, otherwise looks great. Also needs a rebase.

File: 2022/en/src/K05-inadequate-logging.md (hunk 1 above)
Comment (gmaran23) [reply 2022/en/src/K05-inadequate-logging.md:6]: It was supposed to mean the logs in read were either not recorded, or monitored. Your perspective is correct. We will improve this GIF 'slightly' to mean 'inadequate' logging and monitoring.

File: 2022/en/src/K05-inadequate-logging.md (hunk 2 above)
Comment (gmaran23) [reply 2022/en/src/K05-inadequate-logging.md:40]: This is an example Audit Log highlighting the source, destination, timestamps, and other details. We will add a note to include these details.

File: 2022/en/src/K06-broken-authentication.md (hunk 3 above)
Comment (gmaran23) [reply 2022/en/src/K06-broken-authentication.md:4]: Open ID Connect. This is a typo that needs fixing.

File: 2022/en/src/K06-broken-authentication.md (hunk 4 above)
Comment (gmaran23) [reply 2022/en/src/K06-broken-authentication.md:34]: There is a missing 'not' in "Service account tokens should **not** be used from outside the cluster". Great catch!

File: 2022/en/src/K05-inadequate-logging.md (hunk 1 above)
Comment (gmaran23) [reply 2022/en/src/K05-inadequate-logging.md:6]: The illustration is now better in representing the actual idea of _inadequate_ logging, and monitoring. Is that good?

File: 2022/en/src/K05-inadequate-logging.md (hunk 2 above)
Comment (gmaran23) [reply 2022/en/src/K05-inadequate-logging.md:40]: The illustration now has a text up top highlighting what the sample event log is. Is that good?

File: 2022/en/src/K06-broken-authentication.md (hunk 3 above)
Comment (gmaran23) [reply 2022/en/src/K06-broken-authentication.md:4]: OIDC. Fixed.

File: 2022/en/src/K06-broken-authentication.md (hunk 4 above)
Comment (gmaran23) [reply 2022/en/src/K06-broken-authentication.md:34]: Recommendations are not inline with the markdown text. Marking as resolved.

**Comment by gmaran23** [comment]
@szh Thanks for the review, and comments. All requested changes incorporated, and rebased as well.

**Review by szh — APPROVED** [review]
LGTM! Thank you so much for creating these.

**Pull request merged**